packages/dev/core/src/Meshes/meshUtils.ts

```diff
@@ -216,3 +217,25 @@ export function computeMaxExtents(
 
     return maxExtents;
 }
+
+/**
+ * Removes unreferenced vertex data from the given meshes.
+ * This is useful for cleaning up unused vertex data, such as UV sets, to reduce memory usage and stay under graphics device limits.
+ * @param meshes The array of meshes to clean up.
+ */
+export function RemoveUnreferencedVerticesData(meshes: readonly AbstractMesh[]) {

```

Comment:
This function only removed unused UV vertices data. Should it also remove other types?

Reply:
Maybe? I kept the name somewhat generic so we could add more in the future. But I could change it to be more specific, and if we want more in the future we introduce a more generic function. I'm ok either way.

Reply:
Maybe mark the function as experimental and document the fact that it only removes UVs for now?